ui/src/components/Feed/FeedPost.svelte

```diff
@@ -59,17 +58,22 @@
       return api.portal.do.subscribe(key);
     }
 
-    // This is a little confusing but we're merging the global list of comments
-    // with any comments that we have made ourselves on the post, which should
-    // mean that our comment shows up instantly even if our connection to the
-    // indexer is not good
-    replies = [...(getReplies(key) || []), ...(getRepliesByTo(me, key) || [])]
-      .filter((a, i, arr) => {
-        return (
-          i === arr.findIndex((i) => keyStrFromObj(i) === keyStrFromObj(a))
-        );
-      })
-      .sort((a, b) => fromUrbitTime(a.time) - fromUrbitTime(b.time));
+    // recursively fetch replies so we can display the total number of replies per OP
+    const getNestedReplies = (replies) => {
+      if (Array.isArray(replies) && replies.length > 0) {
+        for (let reply of replies) {
+          if (!allReplies.has(reply)) {
+            allReplies.add(reply);
+            const repliesToReply = getReplies(reply);
+            if (repliesToReply) {
+              getNestedReplies(repliesToReply);
+            }
+          }
+        }
+      }
+    };
+
+    getNestedReplies(getReplies(key));
```

Comment:
this will probably introduce lag after you submit your own comment sometimes. i would recommend changing this line to
```
getNestedReplies([...getReplies(key), ...(getRepliesByTo(me, key)]);
```

Reply:
pretty sure initially showing 0s was happening before this PR.

Reply:
you're right tho, now it shows 0s indefinitely